librz/asm/arch/amd29k/amd29k.c

```diff
@@ -441,12 +449,7 @@ bool amd29k_instr_decode(const ut8* buffer, const ut32 buffer_size, amd29k_instr
 			instruction->mnemonic = in->mnemonic;
 			instruction->op_type = in->op_type;
 			return true;
-		} else if (in->cpu[0] != '*' && in->cpu[3] == '0' && in->mask == buffer[0]) {
-			in->decode (instruction, buffer);
-			instruction->mnemonic = in->mnemonic;
-			instruction->op_type = in->op_type;
-			return true;
-		} else if (in->cpu[0] != '*' && in->cpu[3] == '5' && in->mask == buffer[0]) {
+		} else if (in->cpu[0] != '*' && in->mask == buffer[0] && is_cpu(cpu, in)) {
```

Comment:
I don't know about this arch, but is this condition really the same as the ones that were deleted?

Reply:
actually that was bugged. i wrote the code so when i noticed what i forgot, i blame myself for writing something this stupid.